index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <!-- Header section -->
+    <header class="header">
+      <nav class="nav-bar">
+        <h3>Ha Ell</h3>
+        <img src="./image_icons/Icon-Menu.svg" alt="bar icon"/>
+      </nav>
+    </header>
+    <!-- End of Header section -->
+    <!-- Headline section -->
+    <section class="headline-section">
+      <div class="container">
+        <div class="headline-title">
+          <h1>hey there.</h1>
+          <h1>I am Hamza</h1>
```

Comment:
- [ ] Every single page only needs one `h1` tag for SEO. Refactor the the two lines to one `h1` tag.